src/forms/components/FormMembers.tsx

```diff
@@ -49,6 +49,21 @@ export const FormMembers: React.FC<Props> = (props) => {
     setFormMembers(fm);
   }
 
+  const handleEmailAccessChange = (event: React.ChangeEvent<HTMLInputElement>, personId: string) => {
```

Comment:
`handleActionChange` from the same file (just above this function) also does sort of the same thing so why don't we make a function that handles both conditions. 

The way I would achieve is: 
Currently the `handleActionChange` is called like this
```
handleActionChange(fm.memberId, "admin")
```

But I would make it more robust so I can simply call it like this `handleActionChange(fm.memberId, {action: "admin"})`. What this does is, now I can replace your `handleEmailAccessChange` call to `handleActionChange(dm.memberId, {emailAccess: true})` thereby eleminating this `handleEmailAccessChange` itself